src/components/RadioButton.js

```diff
@@ -0,0 +1,61 @@
+const React = require('react');
+
+const StyleConstants = require('../constants/Style');
+
+const RadioButton = React.createClass({
+  propTypes: {
+    activeButtonStyle: React.PropTypes.object,
+    buttonStyle: React.PropTypes.object,
+    checked: React.PropTypes.bool,
+    onClick: React.PropTypes.func,
+    style: React.PropTypes.object
+  },
```

Comment:
Excellent. Added a color prop. Basically a shortcut to the activeStyle.color which would be the piece that is overridden most often. Thanks @iheartrachie.